bin/pyenv-installer

```diff
@@ -50,15 +52,78 @@ checkout "${GITHUB}/pyenv/pyenv-update.git"     "${PYENV_ROOT}/plugins/pyenv-upd
 checkout "${GITHUB}/pyenv/pyenv-virtualenv.git" "${PYENV_ROOT}/plugins/pyenv-virtualenv"  "master"
 checkout "${GITHUB}/pyenv/pyenv-which-ext.git"  "${PYENV_ROOT}/plugins/pyenv-which-ext"   "master"
 
-if ! command -v pyenv 1>/dev/null; then
-  { echo
-    colorize 1 "WARNING"
-    echo ": seems you still have not added 'pyenv' to the load path."
-    echo
+write_source() {
+  {
+    echo "Appending the following lines to $1:"
+    echo '  # pyenv'
+    echo '  export PATH="'"$PYENV_ROOT/bin:"'$PATH"'
+    echo '  eval "$(pyenv init --path)"'
```

Comment:
The suggested default is `pyenv init -`

Reply:
While this is true, I've had the most success across multiple OS'es using the above, or the above in conjunction with `pyenv init -`. Maybe that's an issue for a separate thread, though? Are there any disadvantages to including `pyenv init --path`?

Reply:
Yes. Big disadvantage. That's not the suggested setup. :slightly_smiling_face:
The suggested setup is not random. It was worked out based on user input about their use cases, to provide full features, in the vast majority of cases and with minimal side effects.

E.g. it provides shell completion, and it doesn't add duplicate PATH entires in a nested shell.

Reply:
Sounds good. I'll see if I can replicate some of my issues with this suggested setup.

To be clear, I'm talking about having more success in general with:

```bash
eval "$(pyenv init -)"
eval "$(pyenv init --path)"
```

Over:
```bash
eval "$(pyenv init -)"
```

But maybe this comes down to a debate of where pyenv shims belong in the PATH and of course that will be user-dependent.

Reply:
6cad90a24ac399237e615b13b115e7b433355695